app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1171,10 +1171,31 @@ def test_no_treatment_branches_for_rollout(self):
             is_sticky=True,
             targeting_config_slug=NimbusExperiment.TargetingConfig.MAC_ONLY,
         )
-        self.assertNotEqual(experiment.treatment_branches, [])
+
         experiment.is_rollout = True
         experiment.save()
-        self.assertEqual(experiment.treatment_branches, [])
+
+        for branch in experiment.treatment_branches:
+            branch.delete()
+
+        data = {
+            "application": NimbusExperiment.Application.DESKTOP,
+            "is_sticky": "false",
+            "is_rollout": "true",
+            "targeting_config_slug": NimbusExperiment.TargetingConfig.MAC_ONLY,
+            "treatment_branches": [
+                {"name": "treatment A", "description": "desc1", "ratio": 1},
+                {"name": "treatment B", "description": "desc2", "ratio": 1},
+            ],
+            "changelog_message": "test changelog message",
+            "channel": "",
+        }
+        serializer = NimbusReviewSerializer(
+            experiment, data=data, partial=True, context={"user": self.user}
+        )
+        self.assertFalse(serializer.is_valid())
+        error = serializer.errors["treatment_branches"][0]["name"][0]
+        self.assertEqual(error[0:14], "A rollout may ")
```

Comment:
The API is sending back a constant error message, `NimbusConstants.ERROR_SINGLE_BRANCH_FOR_ROLLOUT`, so can't you just compare to that?